app/main.py

```diff
@@ -1,8 +1,25 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    persons = []
+    for person_data in people:
+        name = person_data["name"]
+        age = person_data["age"]
+        person = Person(name, age)
+        persons.append(person)
+
+    for person_data in people:
+        person = Person.people[person_data["name"]]
```

Comment:
Accessing `Person.people` using `person_data["name"]` assumes that the name exists in the dictionary. Consider adding error handling to manage cases where the name might not be found.